app/components/carousel/component.tsx

```diff
@@ -82,7 +82,7 @@ export const Carousel: React.FC<CarouselProps> = ({ images }: CarouselProps) =>
                 <div
                   className="relative w-full"
                   style={{
-                    paddingBottom: '56.25%',
+                    paddingBottom: '64.61318052%',
```

Comment:
Why did you change this percentage? 56.25 is because 16/9 images. If this change is related to some reason I'm glad to hear it.

Reply:
Done